src/internet.ts

```diff
@@ -426,4 +426,23 @@ export class Internet {
     };
     return _password(len, memorable, pattern, prefix);
   }
+
+  /**
+   * Generates a random emoji.
+   *
+   * @param types A list of the emoji types that should be used.
+   *
+   * @example
+   * faker.internet.emoji() // '🥰'
+   * faker.internet.emoji(['food', 'nature']) // '🥐'
+   */
+  emoji(types?: Array<EmojiType>): string {
```

Comment:
Just to be a bit more future safe and we already try to do this in other places, could we use options here?

```suggestion
  emoji(options: { types?: Array<EmojiType> } = {}): string {
```

These can be extended on demand in the future

Reply:
Done here: https://github.com/faker-js/faker/pull/504/commits/af8f73bf96f5ea5bb58924b75da2829caeb02c56 !